docs/core/compatibility/sdk/9.0/nugetaudit-transitive-packages.md

```diff
@@ -7,6 +7,10 @@ ms.date: 11/14/2024
 
 The [`dotnet restore` command](../../../tools/dotnet-restore.md), which restores the dependencies and tools of a project, now produces security vulnerability warnings for transitive packages by default.
 
+> [!NOTE]
+> This change was reverted in the 9.0.101 SDK.
+> [A modified implementation is being re-introduced in .NET 10](../10.0/nugetaudit-transitive-packages.md).
```

Comment:
@marcpopMSFT @baronfel @dsplaisted  is there existing precedence on how to handle breaking changes that were later reverted?

Reply:
If it was introduced and reverted in the same major version, we usually just delete the file. If you think that's best, then we should add a note/remark to the new .NET 10 breaking change article to describe what happened to the .NET 9 breaking change, in case people are looking for it.

Reply:
The docs compiler is showing an error because there isn't a redirect for the deleted file.   Should I set a redirect from the 9.0 breaking change to the 10.0?

Reply:
Yes, that makes sense. Here's the redirection file: https://github.com/dotnet/docs/blob/main/.openpublishing.redirection.core.json